eng/scripts/Language-Settings.ps1

```diff
@@ -177,7 +178,7 @@ function ValidatePackage($packageName, $packageVersion, $workingDirectory) {
       install `
       $packageExpression `
       --no-cache-dir `
-      --target $installTargetFolder 2>&1
+      --target $installTargetFolder 2>&1 --extra-index-url=$extraIndexFeed
```

Comment:
Do we want to add this an an extra or override the existing pipeline completely?

Reply:
this is only needed for daily update branch. Will change this to run the argument when `extra-index-url` config is set

Reply:
@scbedd here is another potential case of the extra-index-url problem.